Tasks/cURLUploader/task.json

```diff
@@ -0,0 +1,78 @@
+{
+    "id": "AD8974D8-DE11-11E4-B2FE-7FB898A745F3",
+    "name": "cURLUploader",
+    "friendlyName": "cURL Upload Files",
+    "description": "Use cURL to upload files with supported protocols. (FTP, FTPS, SFTP, HTTP, and more)",
+    "category": "Utility",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0 
+    },
+    "demands": [
+        "curl"
+    ],
+    "inputs": [
+        {
+            "name": "files",
+            "type": "filePath",
+            "label": "Files",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "File(s) to be uploaded.  Wildcards can be used. For example, `**\\*.zip` for all ZIP files in all subfolders."
+        },
+        {
+            "name": "username",
+            "type": "string",
+            "label": "Username",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Specify the username to use for server authentication."
+        },
+        {
+            "name": "password",
+            "type": "string",
+            "label": "Password",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Specify the password to use for server authentication.  Password should reference a secret variable defined on the variable tab."
+        },
+        {
+            "name": "url",
+            "type": "string",
+            "label": "URL",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Specify the URL where the file(s) will be uploaded to.  Directory should end with a trailing slash.  Possible protocols include DICT, FILE, FTP, FTPS, GOPHER, HTTP, HTTPS, IMAP, IMAPS, LDAP, LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, TELNET and TFTP."
+        },
+        {
+            "name": "redirectStderr",
+            "type": "boolean",
+            "label": "Redirect Standard Error to Standard Out",
+            "defaultValue": "true",
+            "required": false,
+            "helpMarkDown": "Add '--stderr -' to cURL.  By default cURL writes progress bar to stderr, and build will pick it up as error output. Suppressing this behavior."
+        },
```

Comment:
What would you think about placing this Redirect checkbox in "groupName":"advanced"?  No one would ever really want to change the default value.